DDS-1823 [Link] Fix of QA test (#138)
dsv-rp/DDS: .changeset/ninety-rats-wait.md

---
"@daikin-oss/design-system-web-components": patch
---

> Link: fix issues with the component in the Firefox browser. (DDS-1823)

Reviewer: ```suggestion
Link: Fixed an issue where the color was incorrect in Firefox. (DDS-1823)
```
